plugin.yml

```diff
@@ -63,6 +63,18 @@ configuration:
       type: string
     use-agent-node-affinity:
       type: boolean
+    print-resulting-job-spec:
+      type: boolean
+      default: false
```

Comment:
Interestingly, I couldn't find use of `default` in any Buildkite plugin https://github.com/topics/buildkite-plugin, but JSON Schema that Buildkite uses seems to allow it https://json-schema.org/understanding-json-schema/reference/generic.html#annotations

Reply:
That's interesting, does buildkite actually set a default value from the yaml on behalf of the plugin or does it allow it solely for documentation purposes?